microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -21,33 +21,29 @@ ms.date: 09/13/2023
 
-To refresh your knowledge of the technologies used, see the following articles:
-
-* Use HTML, CSS, and JavaScript to build a web page. See [Getting started with the web](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web)
-
-* Use DevTools to make basic changes to CSS. See [Get started viewing and changing CSS](../css/index.md).
-
-* Run a local HTTP web server. See: 
-
-  * Using Node.js (used for this tutorial): [Set up a localhost server](../../visual-studio-code/microsoft-edge-devtools-extension/install.md#step-6-set-up-a-localhost-server) in _Installing the DevTools extension for Visual Studio Code_.
-  
-  * Using Python: [Running a simple local HTTP server](https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#running_a_simple_local_http_server) in _How do you set up a local testing server?_
+See also:
+* [Getting started with the web](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web) - How to use HTML, CSS, and JavaScript to build a web page. 
+* [Get started viewing and changing CSS](../css/index.md) - How to use DevTools to make basic changes to CSS.
+* [Set up a localhost server](../../visual-studio-code/microsoft-edge-devtools-extension/install.md#step-6-set-up-a-localhost-server) in _Installing the DevTools extension for Visual Studio Code_.  How to run a local HTTP web server by using Node.js.
+* [Running a simple local HTTP server](https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#running_a_simple_local_http_server) in _How do you set up a local testing server?_ - Using Python.
 
 
 <!-- ====================================================================== -->
-## Introduction
+## Overview
+<!-- https://developer.chrome.com/docs/devtools/workspaces#overview -->
 
-A DevTools _workspace_ lets you save changes that you make to a local copy of the source code to the same file on your computer, so that changes are retained across refreshes of the page. Here's a typical scenario for using a workspace:
+A DevTools _workspace_ lets you save changes that you make to a local copy of the source code to the same file on your computer, so that changes are retained across refreshes of the page.  Here's a typical scenario for using a workspace:
 
 * You have the source code for the demo website on your desktop.
 
-* You are running a local web server from the source code directory, so that the site is accessible at `localhost:8080`. Note: If you use the Python server option, the default port number is `8000`.
+* You are running a local web server from the source code directory, so that the site is accessible at `localhost:8080`.  Note: If you use the Python server option, the default port number is `8000`.
```

Comment:
```suggestion
* You are running a local web server from the source code directory, so that the site is accessible at a local development URL such as `http://localhost:8080`.
```